components/PyTorch/Create_fully_connected_network/component.py

```diff
@@ -0,0 +1,43 @@
+from kfp.components import create_component_from_func, OutputPath
+
+def create_fully_connected_pytorch_network(
+    layer_sizes: list,
+    network_path: OutputPath('PyTorchScriptModule'),
+    activation_name: str = 'relu',
+    random_seed: int = 0,
+):
+    '''Creates fully-connected network in PyTorch ScriptModule format'''
+    import torch
+    torch.manual_seed(random_seed)
+
+    activation = getattr(torch, activation_name, None) or getattr(torch.nn.functional, activation_name, None)
+    if not activation:
+        raise ValueError(f'Activation "{activation_name}" was not found.')
+
+    class ActivationLayer(torch.nn.Module):
+        def forward(self, input):
+            return activation(input)
+
+    layers = []
+    for layer_idx in range(len(layer_sizes) - 1):
+        layer = torch.nn.Linear(layer_sizes[layer_idx], layer_sizes[layer_idx + 1])
+        layers.append(layer)
+        if layer_idx < len(layer_sizes) - 2:
```

Comment:
Question: Why is ActivationLayer not added to the last layer when  `layer_idx == len(layer_sizes) - 2`?